### PR TITLE
Editor Completion: the rest of projection scenarios tests

### PR DESCRIPTION
**Pull request by ivanbasov**
Fixes https://github.com/dotnet/roslyn/issues/27446
Test only

### CODE REVIEW

File: src/EditorFeatures/TestUtilities2/Intellisense/ModernCompletionTestState.vb [hunk 1]
```diff
@@ -183,24 +182,33 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
...
             Assert.Equal(expectedOrder.Count, items.Count)
             For i = 0 To expectedOrder.Count - 1
                 Assert.Equal(expectedOrder(i), items(i).DisplayText)
             Next
         End Sub
 
+        Public Overrides Async Function WaitForAsynchronousOperationsAsync(view As ITextView) As Task
+            Await Task.WhenAll(MyBase.WaitForAsynchronousOperationsAsync(), Task.Run(Sub()
+                                                                                         Dim items = GetComputedItemsOpt(TextView)
+                                                                                     End Sub))
```
Comment (jasonmalinowski) [inline src/EditorFeatures/TestUtilities2/Intellisense/ModernCompletionTestState.vb]: What's going on here? Why is this stronger than just waiting for asynchronous operations directly?

Comment (sharwell) [reply src/EditorFeatures/TestUtilities2/Intellisense/ModernCompletionTestState.vb]: This went away in the latest version, where we don't need to separate legacy and modern completion.

**Comment by CyrusNajmabadi** [comment]
@ivanbasov is this PR still applicable?  Do you want to move forward with this?  Or should we close out?

**Comment by CyrusNajmabadi** [comment]
Closing due to lack of movement.

**Pull request closed without merging**